lib/fluent/plugin/out_s3.rb

```diff
@@ -41,6 +41,10 @@ def initialize
       config_param :external_id, :string, default: nil, secret: true
       desc "The region of the STS endpoint to use."
       config_param :sts_region, :string, default: nil
+      desc "A http proxy url for requests to aws sts service"
+      config_param :sts_http_proxy, :string, default: nil, secret: true
+      desc "A url for a regional sts api endpoint, the default is global"
+      config_param :sts_endpoint_url, :string, default: nil ## STS_region
```

Comment:
What does it mean? > `## STS_region`

Reply:
just a comment, it wrote it there to help me figure out what everything means.
I'll remove it with my next push.